e2e/testcases/helm_sync_test.go

```diff
@@ -321,17 +339,21 @@ func TestHelmGCENode(t *testing.T) {
 	nt := nomostest.New(t, nomostesting.SyncSource, ntopts.Unstructured,
 		ntopts.RequireGKE(t), ntopts.GCENodeTest)
 
+	chartName := pushHelmChart(nt, t, privateCoreDNSHelmChart, privateCoreDNSHelmChartVersion)
+	defer func() {
```

Comment:
I'd suggest implementing this with a `nt.T.Cleanup` that's registered inside the `pushHelmChart` function, ideally registered before the chart is pushed so that the cleanup is guaranteed to run even if there is an error

Reply:
Done.